README.rst

```diff
@@ -45,126 +45,80 @@ Once the list of datasets is collated a number of batches must be created:
+    $ python roocs_utils/catalog_maker/cli.py run -p c3s-cmip6 -b 1 -r local
 
 or running all batches on lotus:
 
 .. code-block::
 
-    $ python roocs_utils/inventory/cli.py run -p c3s-cmip6 -r lotus
+    $ python roocs_utils/catalog_maker/cli.py run -p c3s-cmip6 -r lotus
 
-This creates a pickle file containing an ordered dictionary of the inventory for each dataset. It also creates a pickle file for any errors.
+This creates a pickle file containing an ordered dictionary of the entry for each file in each dataset. It also creates a pickle file for any errors.
 
-Viewing records and errors
+Viewing entries and errors
 ==========================
 
 To view the records:
 
 .. code-block::
 
-    $ python roocs_utils/inventory/cli.py list -p c3s-cmip6
+    $ python roocs_utils/catalog_maker/cli.py list -p c3s-cmip6
 
 and to see any errors:
 
 .. code-block::
 
-    $ python roocs_utils/inventory/cli.py show-errors -p c3s-cmip6
-
-To just get a count of how many datasets have been scanned:
-
-.. code-block::
-
-    $ python roocs_utils/inventory/cli.py list -p c3s-cmip6 -c
+    $ python roocs_utils/catalog_maker/cli.py show-errors -p c3s-cmip6
 
-Writing the inventory
-=====================
-
-The final command is to write the inventory to a yaml file. There are 2 options for this.
-
-1.
+To just get a count of how many files have been scanned:
 
 .. code-block::
 
-    $ python roocs_utils/inventory/cli.py write -p c3s-cmip6 -v files
-
-writes the inventory file ``c3s-cmip6-inventory-files.yml`` and includes the file names for each dataset:
+    $ python roocs_utils/catalog_maker/cli.py list -p c3s-cmip6 -c
 
+Writing to CSV
+==============
 
-.. code-block::
-
-    - path: ScenarioMIP/CCCma/CanESM5/ssp370/r1i1p1f1/Amon/rsutcs/gn/v20190429
-      ds_id: c3s-cmip6.ScenarioMIP.CCCma.CanESM5.ssp370.r1i1p1f1.Amon.rsutcs.gn.v20190429
-      var_id: rsutcs
-      array_dims: time lat lon
-      array_shape: 1032 64 128
-      time: 2015-01-16T12:00:00 2100-12-16T12:00:00
-      latitude: -87.86 87.86
-      longitude: 0.00 357.19
-      size: 33845952
-      size_gb: 0.03
-      file_count: 1
-      facets:
-        mip_era: c3s-cmip6
-        activity_id: ScenarioMIP
-        institution_id: CCCma
-        source_id: CanESM5
-        experiment_id: ssp370
-        member_id: r1i1p1f1
-        table_id: Amon
-        variable_id: rsutcs
-        grid_label: gn
-        version: v20190429
-      files:
-      - rsutcs_Amon_CanESM5_ssp370_r1i1p1f1_gn_201501-210012.nc
-
-2.
+The final command is to write the entries to a csv file.
 
 .. code-block::
 
-    $ python roocs_utils/inventory/cli.py write -p c3s-cmip6 -v c3s
+    $ python roocs_utils/catalog_maker/cli.py write -p c3s-cmip6
 
-writes the inventory file ``c3s-cmip6-inventory.yml`` and does not include file names:
+The csv file will be generated in the ``catalog_dir`` specified in ``roocs_utils/etc/roocs.ini`` and will have the name "{project}_{version_stamp}.csv".
```

Comment:
with intake we can have just *one* catalog yaml file with references to *all* project csv files:
https://github.com/cp4cds/c3s_34g_manifests/blob/master/intake/catalogs/c3s.yaml

Reply:
@cehbrecht do you think there is an advantage to having one catalog YAML file for all project CSV files? It would still need updating each time there was a change.

Reply:
We would have only *one* intake catalog (single url pointer). Using intake one can load the project wanted, c3s-cmip6, c3s-cordex, etc.

Reply:
Happy to change this - so each project needs an entry in https://github.com/cp4cds/c3s_34g_manifests/blob/master/intake/catalogs/c3s.yaml like the `c3s-cmip6` one?

Reply:
@cehbrecht nice, I like that.